main/inc/lib/document.lib.php

```diff
@@ -373,6 +445,16 @@ public static function file_send_for_download(
                         }
                     }
                     break;
+				case 'video/mp4':
+				case 'audio/mpeg':
+				case 'audio/mp4':
+				case 'audio/ogg':
+				case 'audio/webm':
+				case 'audio/wav':
+				case 'video/ogg':
+				case 'video/webm':		
+					self::smartReadFile($full_file_name,$filename,$contentType);
+					break;
```

Comment:
Maybe `break` should be `exit;` to skeep the code below

Reply:
Thanks I'll review all those early next week!